blockchain/message_pool/src/msgpool/msg_pool.rs

```diff
@@ -261,9 +267,10 @@ where
         let local_addrs = mp.local_addrs.clone();
         let network_sender = Arc::new(mp.network_sender.clone());
         let network_name = mp.network_name.clone();
+        let republish_interval = 10 * block_delay + PROPAGATION_DELAY_SECS;
         // Reacts to republishing requests
         task::spawn(async move {
-            let mut interval = interval(Duration::from_millis(REPUBLISH_INTERVAL));
+            let mut interval = interval(Duration::from_millis(republish_interval));
```

Comment:
Good point, this is a bit eyebrow-raising yes. In Lotus the interval is coherent and expressed in seconds.
I would think `Duration::from_secs` is more correct here. Let's open a separate issue for investigation (since I didn't changed the logic in this PR).